test/data/upgrade8to9/upgrade_prepare.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This upgrades the systemto latest RHEL-9
+set -xeuo pipefail
+
+# Disable gpgcheck for internal repositories
+echo "gpgcheck=0" >> /etc/yum.repos.d/baseos.repo
+echo "gpgcheck=0" >> /etc/yum.repos.d/appstream.repo
+
+# Install SUT
+dnf install -y osbuild-composer composer-cli
+
+# Prepare the upgrade
+curl -k -o /etc/yum.repos.d/oam-group-leapp-rhel-8.repo https://gitlab.cee.redhat.com/leapp/oamg-rhel8-vagrant/-/raw/master/roles/init/files/leapp-copr.repo
+dnf install leapp-upgrade-el8toel9 -y
+curl -kLO https://gitlab.cee.redhat.com/leapp/oamg-rhel7-vagrant/raw/master/roles/init/files/prepare_test_env.sh
+source /root/prepare_test_env.sh
+get_data_files
```

Comment:
I don't understand what this script/commands are needed for? Doesn't look like an official way to upgrade though. Maybe a link to docs will help.

Shouldn't this leapp command be installed from an official 8.6 distro at least ?

Reply:
This comes directly from the internal guide from the ticket we got, AFAIK the upgrade is not officially released yet, I'll post more information to our Jira ticket